src/server/api/routers/ballot.ts

```diff
@@ -37,9 +36,7 @@ export const ballotRouter = createTRPCRouter({
     .input(BallotSchema)
     .mutation(async ({ input, ctx }) => {
       const voterId = ctx.session.user.name!;
-      if (isAfter(new Date(), config.votingEndsAt)) {
-        throw new TRPCError({ code: "FORBIDDEN", message: "Voting has ended" });
-      }
+
```

Comment:
Why took this out? I think we still need to prevent submit ballot after voting ends? but we need to get the `votingEnds` from poll data